src/elm/Page/Shop.elm

```diff
@@ -553,6 +553,36 @@ viewShopFilterDropdown translations maybeFilter loggedIn =
     viewMenuFilterDropdown ClickedFilter decoder options
 
 
+viewShopFilterTab :  ( String, String, String ) -> Maybe Filter -> LoggedIn.Model -> Html Msg
+viewShopFilterTab translations maybeFilter loggedIn =
+    let
+        ( communities, all, user ) =
+            translations
+
+        decoder =
```

Comment:
@JerryNyoike  I know that is what existed before but I suspect the issue is arising somewhere here, I dont see why you need to go through translations for the tab is it possible to get rid  of this and use a Custom Type instead to see which tab is actually active?

Also there is left over code for the dropdowns perhaps you can remove that too?